.gitignore

```diff
@@ -8,6 +8,7 @@
 *.user
 *.project
 .idea
+.vscode
```

Comment:
would it make sense to have a section about vscode with multiple folders / files in there?

Reply:
Let me remove it from there and we take it from your PR